github/CheckRun.py

```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Dhruv Manilawala <dhruvmanila@gmail.com>                      #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import datetime
+
+import github.CheckRunAnnotation
+import github.CheckRunOutput
+import github.GithubApp
+import github.GithubObject
+import github.PaginatedList
+import github.PullRequest
+
+
+class CheckRun(github.GithubObject.CompletableGithubObject):
+    """
+    This class represents check runs.
+    The reference can be found here https://docs.github.com/en/rest/reference/checks#check-runs
+    """
+
+    def __repr__(self):
+        return self.get__repr__(
+            {"id": self._id.value, "conclusion": self._conclusion.value}
+        )
+
+    @property
+    def app(self):
+        """
+        :rtype: :class:`github.GithubApp.GithubApp`
```

Comment:
Properties have a type, not an rtype.

Reply:
Nice timing lol. I was just going to reply.
Yes, I will change that.